docs/src/architecture/notes.md

```diff
@@ -1,71 +1,71 @@
 
-## Inputs
+### Inputs
 A note script can take parameters (passed via the stack) as inputs.
 
-## Serial number
+### Serial number
 A note's serial number identifies the note and this is needed to create the note's hash and nullifier. The serial number is used to break linkability between note hash and note nullifier.
 
-# Note metadata
-For every note the Miden Node stores metadata in the Note DB. This metadata includes: 
-* user-defined tag (1 field element)
-* sender (1 field element)
-* number of assets contained in the note (1 field element)
-____
+## Note metadata
+For every note, the Miden Node stores metadata in the Notes DB. This metadata includes: 
+* User-defined tag (1 field element)
+* Sender (1 field element)
+* Number of assets contained in the note (1 field element)
 
-# Types of notes
-There are two types of notes in Miden. Notes can be stored privately in the [Notes DB](https://0xpolygonmiden.github.io/miden-base/architecture/state.html#notes-database) with only the note hash. Or notes can be stores publicly with all data.
+## Types of notes
+There are two types of notes in Miden. Notes can be stored privately in the [Notes DB](https://0xpolygonmiden.github.io/miden-base/architecture/state.html#notes-database) with only the note hash or can be stored publicly with all the data.
 
-# Note hash
+## Note hash
 The note hash is computed as: 
 
 `hash(hash(hash(hash(serial_num, [0; 4]), script_hash), input_hash), vault_hash)`
 
 This achieves the following properties:
 - Every note can be reduced to a single unique hash.
-- To compute a note's hash, we do not need to know the note's serial_num. Knowing the hash
-    of the serial_num (as well as script hash, input hash and note vault) is sufficient.
+- To compute a note's hash, we do not need to know the note's serial number. Knowing the hash
+    of the `serial_num` (as well as script hash, input hash and note vault) is sufficient.
 - Moreover, we define `recipient` as: \
     `hash(hash(hash(serial_num, [0; 4]), script_hash), input_hash)`. This allows computing the note hash from recipient and note vault.
 - We compute the hash of `serial_num` as `hash(serial_num, [0; 4])` to simplify processing within
 the VM.
 
-# Note nullifier
-The nullifier is the note's index in the [Nullifier DB](https://0xpolygonmiden.github.io/miden-base/architecture/state.html#nullifier-database). The Nullifier DB stores the information whether a note was already consumed. 
+## Note nullifier
+The nullifier is the note's index in the [Nullifier DB](https://0xpolygonmiden.github.io/miden-base/architecture/state.html#nullifier-database). The Nullifier DB stores the information that whether a note was already consumed or not. 
```

Comment:
Nit: I think in this sentence is something wrong?

`The Nullifier DB stores the information that whether a note was already consumed or not.`

Reply:
But I am not a native speaker

Reply:
LGTM